src/placeos-driver/proxy/scheduler.cr

```diff
@@ -44,11 +55,11 @@ class PlaceOS::Driver::Proxy::Scheduler
     spawn(same_thread: true) { run_now(block) } if immediate
     wrapped = nil
     task = Tasker.at(time) do
-      @schedules.delete(wrapped.not_nil!.object_id)
+      @schedules.delete(wrapped.not_nil!)
       run_now(block)
     end
-    wrap = wrapped = TaskWrapper.new(task, @schedules)
-    @schedules[wrap.object_id] = wrap
+    wrap = wrapped = TaskWrapper.new(task, @callback)
```

Comment:
Is this double assign needed?

Reply:
saves a `.not_nil!` on the `<<` the line below